packages/superset-ui-chart-controls/src/shared-controls/components/ColumnConfigControl/constants.tsx

```diff
@@ -33,12 +33,22 @@ export type SharedColumnConfigProp =
   | 'colorPositiveNegative'
   | 'columnWidth'
   | 'fractionDigits'
+  | 'emitTarget'
   | 'd3NumberFormat'
   | 'd3SmallNumberFormat'
   | 'd3TimeFormat'
   | 'horizontalAlign'
   | 'showCellBars';
 
+const emitTarget: ControlFormItemSpec<'Input'> = {
+  controlType: 'Input',
+  label: t('Emit Target'),
+  description: D3_FORMAT_DOCS,
```

Comment:
We should add a proper description here. Something like `t('If you wish to specify a different target column than the original column, it can be entered here')`.